generator/src/pkg/service/util.go

```diff
@@ -46,13 +46,19 @@ const (
 	SvcThreadsDefault        = 2
 	SvcReadinessProbeDefault = 5
 
-	EpNamePrefix                = "/end"
-	EpCPUConsumptionDefault     = 0.003
-	EpNetworkConsumptionDefault = 0.002
-	EpMemoryConsumptionDefault  = 0.003
-	EpForwardRequests           = "asynchronous"
+	EpNamePrefix            = "end"
+	EpExecModeDefault       = "sequential"
+	EpNwResponseSizeDefault = 512
+
+	EpExecTimeDefault = "1s"
```

Comment:
The default values should be also reduced to the a reasonable scale for a single request.